### PR TITLE
Support for SCL proxy

### PR DESCRIPTION
**Pull request by lzap**
This is a WIP branch, but it builds fine and it works fine.

### CODE REVIEW

File: 00-repos-centos7.ks [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
 repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
-repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/nightly/el7/$basearch/
-repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/nightly/el7/$basearch/
+repo --name=centos-sclo --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
+#repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/nightly/el7/$basearch/
+#repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/nightly/el7/$basearch/
+# FIXME these are temporary
+repo --name=foreman-rails-el7 --baseurl=https://yum.theforeman.org/rails/foreman-nightly/el7/$basearch/
+repo --name=foreman-el7-koji --baseurl=http://koji.katello.org/releases/yum/foreman-nightly/RHEL/7/x86_64/
```
Comment (evgeni) [inline 00-repos-centos7.ks]: the koji ones should not be needed anymore, everything is published on yum.tfm.o by now.

File: 00-repos-centos7.ks [hunk 2]
```diff
@@ -1,5 +1,10 @@
 repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
 repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
-repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/nightly/el7/$basearch/
-repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/nightly/el7/$basearch/
+repo --name=centos-sclo --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```
Comment (evgeni) [inline 00-repos-centos7.ks]:
```suggestion
repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```

I agree with @ekohl, let's add `-rh` suffix to the name

File: 00-repos-centos7.ks (hunk 1 above)
Comment (lzap) [reply 00-repos-centos7.ks]: Yes I just noticed. Removed.

File: 00-repos-centos7.ks (hunk 2 above)
Comment (lzap) [reply 00-repos-centos7.ks]: There repositories do not make it to the image, it's used only during the livecd creation. But I can change it.

**Comment by lzap** [comment]
Thanks!

**Pull request merged**